### DISCUSSION TITLE
What's the purpose of the WINE_FIXME("Unverified!\n") debug outputs?

### DISCUSSION THREAD

**Discussion by Volkanite** [AndreRH/hangover#132] [General]
Just a curious question. What's the purpose of the `WINE_FIXME("Unverified!\n")` debug lines? What do they mean? What exactly is unverified about the code? Specifically, those belonging to the file [exception.c](https://github.com/AndreRH/hangover/blob/d793c90c6d1a6f95f68b4778ca6a93a1f0007367/dlls/ntdll/exception.c#L94)

**Comment by stefand** [comment]
It means those functions were generated by an automated script and didn't have any kind of manual review to see if the parameters and return values can be passed 1:1 from x86 to the arm side. Most of the time it is harmless when going from x86_64 to arm64, but 32 bit x86 to arm64 needs quite a lot of fixup.

exception.c is tricky. Most exception handling in hangover is pretty much a rewrite or copypaste of the x86 wine code into the guest side of hangover.

For future development plans have a look at some of my comments on other bug reports re Wine's ongoing 32on64 work.

**Comment by AndreRH** [comment]
This is obsolete now since hangover 0.8.1
